src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,111 @@
+package duke.command;
+
+import duke.TaskList;
+import duke.Ui;
+import duke.Storage;
+import duke.exception.DescriptionFormatException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDos;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class AddCommand extends Command {
+    private String com;
+    private static final DateTimeFormatter DATE_FORMAT_INP = DateTimeFormatter.ofPattern("yyyy-MM-dd' 'HH:mm");
+    private static final DateTimeFormatter DATE_FORMAT_OUT = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+
+    /**
+     * Constructor for AddCommand.
+     *
+     * @param c command in String format.
+     */
+    public AddCommand(String c) {
+        super(0);
+        com = c;
+    }
+
+    @Override
+    public void execute(TaskList tL, Ui ui, Storage st) {
+        String[] inputs = com.split(" ");
+        String[] name;
+        try {
+            switch (inputs[0]) {
+            case "event":
+                if (inputs.length == 1) {
+                    throw new DescriptionFormatException("Wrong format!, please use this format: "
+                            + Event.getFormat());
+                } else {
+                    name = com.split("event ");
+                    String[] desFromTo = name[1].split(" /from ");
+                    String[] fromTo = desFromTo[1].split(" /to ");
+                    if (desFromTo.length != 2 || fromTo.length != 2) {
+                        throw new DescriptionFormatException("Wrong format!, please use this format: "
+                                + Event.getFormat());
+                    }
+                    LocalDateTime ldtF = LocalDateTime.parse(fromTo[0], DATE_FORMAT_INP);
+                    LocalDateTime ldtT = LocalDateTime.parse(fromTo[1], DATE_FORMAT_INP);
+                    String out = tL.addTask(new Event(desFromTo[0], ldtF, ldtT));
+                    ui.showMessage(out);
+                    st.write(tL.getList());
+                }
+                break;
+            case "todo":
```

Comment:
I believe it would be better if theres a space between if and (. I say this appearing in some other if-else statements as well.

Reply:
Yeah, I believe that's right too. I tried to do it like that but some of it was still missed. Thanks for noticing!